protocol_versioned_docs/version-gemini-1b-2022-june-10/farm/farming.mdx

```diff
@@ -0,0 +1,608 @@
+  ```
+  5. We will then open another terminal, change to the downloads directory, then start the farmer node with the following command:
+  ```bash
+  # Replace `WALLET_ADDRESS` below with your account address from Polkadot.js wallet
+  # Replace `PLOT_SIZE` with plot size in gigabytes or terabytes, for instance 100G or 2T (but leave at least 10G of disk space for node)
+  ./subspace-farmer-macos-x86_64-gemini-1b-2022-june-10 farm --reward-address WALLET_ADDRESS --plot-size PLOT_SIZE
+  ```
+  6. It may prompt again in here. Refer to the note on step 4.
+
+  7. You should see something similar in the terminal:
+  ```
+  2022-06-02T00:02:04.552120Z  INFO subspace_farmer::commands::farm: Connecting to node at ws://127.0.0.1:9944
+  2022-06-02T00:02:04.557220Z  INFO jsonrpsee_client_transport::ws: Connection established to target: Target { sockaddrs: [], host: "127.0.0.1", host_header: "127.0.0.1:9944", _mode: Plain, path_and_query: "/" }
+  2022-06-02T00:02:04.575669Z  INFO subspace_farmer::commands::farm: Opening object mapping
+  2022-06-02T00:02:04.617553Z  INFO subspace_farmer::multi_farming: Opening plot
+  2022-06-02T00:02:04.645422Z  INFO subspace_farmer::multi_farming: Opening commitments
+  2022-06-02T00:02:04.666925Z  INFO subspace_farmer::farming: Subscribing to slot info notifications
+  2022-06-02T00:02:04.688218Z  INFO subspace_farmer::archiving: Subscribing to archived segments
+  2022-06-02T00:02:04.689129Z  INFO subspace_farmer::commands::farm: WS RPC server listening on 127.0.0.1:9955
+  2022-06-02T00:02:09.492868Z  INFO subspace_farmer::archiving: Plotted segment segment_index=0
+  2022-06-02T00:02:09.492868Z  INFO subspace_farmer::archiving: Plotted segment segment_index=1
+  2022-06-02T00:02:09.492868Z  INFO subspace_farmer::archiving: Plotted segment segment_index=2
+  2022-06-02T00:02:09.492868Z  INFO subspace_farmer::archiving: Plotted segment segment_index=3
+  2022-06-02T00:02:09.492868Z  INFO subspace_farmer::archiving: Plotted segment segment_index=4
+  2022-06-02T00:02:09.492868Z  INFO subspace_farmer::archiving: Plotted segment segment_index=5
+  ```
+
+  8. You are now Farming! Just wait for your node to fully sync, and for your farmer to fully plot.
+
+</TabItem>
+
+<TabItem value="linux" label="🐧Linux">
+  <div className={styles.buttons}>
+    <Link
+      className="button button--secondary button--lg"
+      to="https://github.com/subspace/subspace/releases/download/gemini-1b-2022-june-10/subspace-node-ubuntu-x86_64-gemini-1b-2022-june-10">
+      1. Download Node Executable
+    </Link>
+    <Link
+      className="button button--secondary button--lg"
+      to="https://github.com/subspace/subspace/releases/download/gemini-1b-2022-june-10/subspace-farmer-ubuntu-x86_64-gemini-1b-2022-june-10">
+      2. Download Farmer Executable
+    </Link>
+  </div>
+  <p>
+
+
+  </p>
+
+  
+  <details>
+    <summary>
+    Expand for Arm/Raspberry Pi Images
```

Comment:
The reason is that they'll not work on older Raspberry PI or systems that aren't 64-bit:
```suggestion
    Expand for Aarch64/64-bit Raspberry Pi Executables
```